### ISSUE TITLE
pg >= 1.3.0

### ISSUE THREAD

**Issue by Nerian** [lsylvester/active_storage-postgresql#16]
Hi,

I ran into this error when trying to update the pg gem

```
ActiveRecord::StatementInvalid:
        PG::UnableToSend: unexpected response from server; first received character was "V"
```

I see we lock the pg gem https://github.com/lsylvester/active_storage-postgresql/commit/5a04f105061d5f37ac0c05d1d15decdf4b32f602

But is this something we can fix so we can update? Did you figure out what the issue/blocker was?

I found these links that may be of help.
* https://github.com/ged/ruby-pg/issues/478
* https://github.com/ged/ruby-pg/issues/460

Thanks

**Comment by larskanis** [comment]
This should be fixed in pg-1.4.6. PR #17 opens the pg gem restriction accordingly.

**Comment by Nerian** [comment]
Thanks 🙏
